vector/src/main/res/values-de/strings.xml

```diff
@@ -2626,6 +2626,7 @@ Verwahre deinen Wiederherstellungsschlüssel an einem sehr sicheren Ort wie eine
     <string name="settings_call_show_confirmation_dialog_summary">Bitte um Bestätigung, bevor du einen Anruf tätigst</string>
     <string name="bottom_sheet_setup_secure_backup_submit">Konfiguration</string>
     <string name="no_permissions_to_start_conf_call">Dir fehlt die Berechtigung in diesem Raum eine Konferenz zu starten</string>
+    <string name="no_permissions_to_start_webrtc_call">Dir fehlt die Berechtigung in diesem Raum einen Anruf zu starten</string>
```

Comment:
Normally translations comes from Weblate. To avoid merge conflict, PR on Element Android can add new default strings, but should not add translations. Could you remove this change please?

Reply:
(See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#internationalisation but I should maybe be more clear on https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#i-want-to-submit-a-pr-to-fix-an-issue)